changelogs/fragments/82580_constructed.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "ansible-doc - Adding a note that only group_vars of explicit groups are loaded (https://github.com/ansible/ansible/pull/82580)."
```

Comment:
that seems like you are adding a feature to the `ansible-doc` CLI, it shoudl read `constructed inventory plugin` instead.

Reply:
Fixed it, thanks for your help @bcoca, really appreciate it!